src/main/resources/static/js/events/script.js

```diff
@@ -97,6 +97,21 @@ $(document).ready(function () {
         }
     });
 
+    const externalProductUrlField = document.getElementById("externalProductUrl");
+
+    function validateExternalProductUrlField() {
+        let products = $("#products");
+        let productsSize = products.children().length;
+
+        if (productsSize > 0 && externalProductUrlField.value !== "") {
```

Comment:
An empty string is equal to false in javascript

Reply:
So do you recommend changing it to
```javascript
if (productsSize > 0 && externalProductUrlField.value)
```

Isn't the way it is now easier to read?

Reply:
That's why we love javascript :D

Reply:
Ah yeah. I might be a bit too stern on this one. I would suggest doing whatever is most clear for the person that will work with this the next time. Following that mantra having the whole thing written out might be better. Apologies for the comment

Reply:
The entire concept of having a falsy option still saddens me as a whole.

Reply:
Might I however suggest that you do a valid url checking? I am usually not a fan of regular expressions, but a well commented one here might prevent future boards from frustration when they use the wrong string. Or maybe render an <a> tag under it stating that this is where it will lead the users. both of these I would see as nice to haves. So if you do not feel like adding this let me know and I will merge it for you :)

Reply:
And there is a piece of text under the field that states:
"Replaces the products with a button to this URL. URL must start with <b>https://</b>. Keep empty if you want to use the products."